src/ontology/edits/oeo-shared.omn

```diff
@@ -1934,10 +1948,15 @@ move to oeo-shared
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956
 rework module structure 
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An economic value is a quantity value that is economically relevant.",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
+obsoletion
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1917
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1931",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "OBSOLETE: An economic value is a quantity value that is economically relevant.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "wirtschaftlicher Wert"@de,
-        rdfs:label "economic value"@en
+        rdfs:comment "This term was made obsolete because its definition is rather vague and thus not useful.",
```

Comment:
```suggestion
        rdfs:comment "This term was made obsolete together with economic value because its definition is rather vague and thus not useful.",
```